feature-detects/mediastream.js

```diff
@@ -0,0 +1,13 @@
+define(['Modernizr'], function (Modernizr) {
+    Modernizr.addTest('mediarecorder', function(){
+        var isMediaRecorderSupported = false;
+
+        try {
+            MediaRecorder;
+            isMediaRecorderSupported = true;
+        } catch (err) {
+            console.log("no MediaRecorder");
+        }
+        console.log(isMediaRecorderSupported);
```

Comment:
We do not use `console.log`s in Modernizr (usually). Instead of returning whether it is compatible in a `console.log` we use `Modernizr.addTest` (the one that it is at the start of the test)